vscode-extension/src/extension.ts

```diff
@@ -495,17 +496,13 @@ async function shareAIConfig(
   const fileName: string = activeEditor.document.fileName;
   const sanitizedFileName: string = sanitizeFileName(fileName);
 
-  // TODO: Add back once CORS is resolved
-  // const policyResponse = await fetch(
-  //   "https://lastmileai.dev/api/upload/publicpolicy"
-  // );
-  // const policy = await policyResponse.json();
   const uploadUrl = "https://s3.amazonaws.com/lastmileai.aiconfig.public/";
   const randomPath = Math.round(Math.random() * 10000);
   const uploadKey: string = `aiconfigs/${getTodayDateString()}/${randomPath}/${sanitizedFileName}`;
-
-  // TODO: Will also need to check for yaml files and change the contentType accordingly
-  const contentType = "application/json";
+  const contentType =
+    getModeFromDocument(activeEditor.document) === "json"
+      ? "application/json"
+      : "application/yaml";
```

Comment:
```suggestion
  const contentType = `application/${getModeFromDocument(activeEditor.document)}`;
```
nit

Reply:
I didn't want to do this in case someday we update `getModeFromDocument()` to also return `"yml"`